core/src/main/java/io/questdb/cairo/wal/seq/SequencerMetadataService.java

```diff
@@ -49,6 +50,13 @@ public void addColumn(
         metadata.addColumn(name, type);
     }
 
+    @Override
+    public void renameTable(CharSequence oldName, CharSequence newName) {
+        assert Chars.equals(oldName, metadata.getTableToken().getTableName());
```

Comment:
case insensitivity

Reply:
done, converted to case insensitive check, added tests